File: Assets/Tank/Panther/AIMovement/MoveTurret_ai.cs
using UnityEngine;

public class MoveTurret_ai : MonoBehaviour {
    public float speed = 30;

    // Turn Right
    void turretRight(){
        transform.Rotate(new Vector3(0, speed * Time.deltaTime, 0));
    }
    // Turn Left
    void turretLeft() {
        transform.Rotate(new Vector3(0, -speed * Time.deltaTime, 0));
    }
}

File: Assets/Tank/Panther/AIMovement/MoveTurret_ai.js
#pragma strict

var speed : float = 30;

	// Turn Right
function turretRight(){
	transform.Rotate(Vector3(0, speed * Time.deltaTime, 0));
}
 	// Turn Left
function turretLeft() {
	transform.Rotate(Vector3(0, -speed * Time.deltaTime, 0));
}